templates/umap/summary.html

```diff
@@ -0,0 +1,13 @@
+<div class="ui container summary">
+  <h1 class="ui header">
```

Comment:
I see 2 and 4 spaces indentations depending on the files, we need to use only one ;)
For HTML, it's usually 2, but I'm OK with any choice.